### PR TITLE
Clipboard button copies $ on install conjur page

### LINKED ISSUE #406: Clipboard button copies $ on install conjur page
The problem is that using `` ```sh-session`` to define the shell block does not set the `data-lang` attribute of the resulting HTML. This causes the block to be treated as a plain-text when performing the clipboard copy and the `$` is included.

### PR DESCRIPTION
**Pull request by jtuttle**
Closes #406 

#### Test steps
1) Set up docs container on localhost using README instructions
1) Confirm that copying from shell blocks on install conjur page with clipboard button does not include `$` characters.

### CODE REVIEW

File: docs/get-started/install-conjur.md [hunk 1]
```diff
@@ -21,24 +21,24 @@ official Conjur containers on DockerHub.
 
 1. In your terminal, download the Conjur quick-start configuration:
 
-   ```sh-session
-   $ curl -o docker-compose.yml https://www.conjur.org/get-started/docker-compose.quickstart.yml
-   ```
+{% highlight shell %}
```
Comment (ryanprior) [inline docs/get-started/install-conjur.md]: Is this a stopgap? eg should we never use "sh-session" fences in documentation, or can we introduce code to make clipboard work correctly with those?

**Comment by ryanprior** [comment]
the test instructions say to set up the docs container following the README, but the files changed don't touch the readme or the docs container- did you miss something?

File: docs/get-started/install-conjur.md (hunk 1 above)
Comment (jtuttle) [reply docs/get-started/install-conjur.md]: Yes, you can consider this a stopgap for now, just a quick way to fix it as it's urgent.

**Comment by jtuttle** [comment]
There are no changes to the README, I only referenced it as the place where you can find instructions for setting up a test container with which to test my install page changes.

**Pull request merged**